test/functional/interface_usdt_utxocache.py

```
@@ -357,16 +357,17 @@ def test_flush(self):
         # that the handle_* functions succeeded.
         EXPECTED_HANDLE_FLUSH_SUCCESS = 3
         handle_flush_succeeds = 0
-        possible_cache_sizes = set()
-        expected_flushes = []
+        expected_flushes = list()
```

Ruff (0.14.6)

Unnecessary `list()` call (rewrite as a literal)

Rewrite as a literal

(C408)